jekyll/_cci2/how-to-override-config.adoc

```diff
@@ -0,0 +1,344 @@
+          paths:
+            - ./node_modules
+      - run:
+          name: "Run team-specific tests"
+          command: npm run test:integration
+----
+
+Notice that this file:
+
+* Does NOT include `version: 2.1`.
+* Does NOT include `orbs:` or `workflows:` sections.
+* Contains only the `jobs:` section with the specific job definitions the team wants to override.
+
+=== 3. Configure your URL allow list
+To use URL-based orbs for team configurations, configure your organization's allow-list settings to include the URL prefix to match the orb file:
+
+#ADD LINK TO ALLOW-LIST HOW-TO WHEN READY#
+
+[source,text]
+----
+https://raw.githubusercontent.com/
+This allows importing configurations from any repository in your GitHub organization.
+----
+
+=== 4. Set up pipeline definitions
+For each project using the centralized configuration the platform team needs to:
+
+* Configure the project to use GitHub App integration. The CircleCI GitHub App integration is required to use a config source outside the project repo. See the xref:github-apps-integration#[GitHub App integration] guide for more information.
+* Create a pipeline definition pointing to the central template configuration. This means setting up a pipeline in which the config source is outside the project repository. The config source will be the centralized, _template_ configuration managed by the platform team.
+
+[#transitioning-to-centralized-configs]
+== Transitioning to centralized configs
+Follow these steps to migrate existing projects to a centralized configuration model:
+
+. *Create central configuration repository:* Copy an existing project configuration to a central repository that will serve as your template.
+. *Setup pipeline definitions:* Create a pipeline definition that uses the central config as its source.
+. *Remove duplicate workflows:* Remove workflow definitions from individual project repositories as they are now defined in the central config.
+. *Remove duplicate job definitions:* Remove standard job definitions from project repositories, keeping only those that need customization.
+. *Implement overrides:* For jobs requiring customization, implement the override mechanism using the steps outlined above.
+
+[#troubleshooting]
+== Troubleshooting
+
+=== Job not being overridden
+If your job is not being overridden as expected, try the following:
+
+* Verify the orb is correctly referenced in your configuration.
+* Check that the job name in the orb matches exactly what you specified in override-with.
+* Ensure the URL for URL-based orbs is in your organization's allow list.
+* Confirm the orb file exists at the specified URL.
+
+=== Pipeline parameter issues
+If using pipeline parameters with URL-based orbs, check the following:
+
+* Remember that pipeline parameters are only provided when triggering a pipeline via the API.
+* For VCS triggers, ensure you are using pipeline values that are available at runtime.
```

Comment:
For VCS triggers, one example we can provide to construct the Orb URL is:
```
https://raw.githubusercontent.com/<< pipeline.git.repo_owner >>/<< pipeline.git.repo_name >>/refs/heads/main/path/to/file/orb.yml
```
Then the pipeline values will be compiled to the team's repo that triggered the change accordingly, and fetch that team's override.

Of course this assumes the org is using Github App, and all the teams' repos are under the same github org.